waspc/src/Wasp/AppSpec/Valid.hs

```diff
@@ -165,7 +165,7 @@ validateOnlyEmailOrUsernameAndPasswordAuthIsUsed spec =
 validateDbIsPostgresIfPgBossUsed :: AppSpec -> [ValidationError]
 validateDbIsPostgresIfPgBossUsed spec =
   [ GenericValidationError
-      "The database provider in the schema.prisma file must be \"postgresql\" since there are jobs with executor set to PgBoss."
+      ("The database provider in the schema.prisma file must be \"" ++ AS.Db.providerToString AS.Db.PostgreSQL ++ "\" since there are jobs with executor set to PgBoss.")
```

Comment:
Ok, so this is first usage site. I can see there that this string must be exact the same string that Prisma uses to specify postgre as a provider. So you are assuming here that `providerToString` returns Prisma keyword/identifier for Postgre. But that function, `providerToString`, in no way offers that guarantee, be it via implemention, types or docs. So that is a future bug waiting to happen.